kong/db/init.lua

```diff
@@ -130,8 +130,16 @@ function DB:reset()
 end
 
 
-function DB:truncate()
-  return self.connector:truncate()
+function DB:truncate(table_name)
+  if table_name ~= nil and type(table_name) ~= "string" then
+    error("table_name must be a string", 2)
+  end
+
+  if table_name then
+    return self.connector:truncate_table(table_name)
+  else
+    return self.connector:truncate()
+  end
```

Comment:
Could we avoid nesting the branch if our previous branch already returns, as usual?